src/anndata_to_avro.py

```diff
@@ -209,8 +209,17 @@ def default(self, obj):
             return json.JSONEncoder.default(self, obj)
 
     def ingest_generator():
+        # Some `uns` metadata has extremely large values that frustrate extract. Cap the allowed size and substitute
+        # a `None` value in these cases.
+        metadata_limit = 2 ** 20
```

Comment:
Any particular reason for this cap?

Reply:
tl;dr big but not too big. I added some prose rationalizing the choice.